index.html

```diff
@@ -1 +1,39 @@
-<!-- JavaScript課題①-2 -->
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="ja">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>task1-2</title>
+</head>
+<body>
+  <script>
+    
+    while(true){
+      let quiz = prompt('日本の首都は？');
```

Comment:
再代入をしないので `const` の方がより適切かと思いました。

```suggestion
      const quiz = prompt('日本の首都は？');
```